src/Kunstmaan/AdminListBundle/AdminList/ExportList.php

```diff
@@ -14,10 +15,16 @@ class ExportList implements ExportableInterface
     public function __construct(ExportListConfiguratorInterface $configurator)
     {
         $this->configurator = $configurator;
+        $this->configurator->buildFilters();
```

Comment:
The travis build is failing on this one, I think you need to add buildFilters method in the ExportListConfiguratorInterface